### ISSUE TITLE
About page is not working

### ISSUE THREAD

**Issue by nishtha53** [Ankit-11525/e-commerce_redstore.github.io#17]
About should have some content on a separate page. I would like to work on it.

**Comment by Ankit-11525** [comment]
Go Ahead !!

**Comment by Ankit-11525** [comment]
Also add contributers information (mainly logo with link) on that page ,if possible

**Comment by harikesh409** [comment]
I suggest using API https://api.github.com/repos/Ankit-11525/e-commerce_redstore.github.io/contributors, to fetch the contributors so the list can be dynamic.

**Comment by nishtha53** [comment]
yes @harikesh409  this is good idea to keep dynamic.

**Comment by Ankit-11525** [comment]
Please try this and add this in our about section !!
About page be like : 
-some detailed information about website 
-all contributers list 
-any remark
-at last footer

**Comment by nishtha53** [comment]
okay sure @Ankit-11525

**Comment by anuragh2002** [comment]
Hey @Ankit-11525 I would like to frame professional-grade content for About page. Like your vision, mission, why you started, founder's note and many more. Can you please co-assign this issue to me for the content part if @nishtha53 is comfortable with it?

**Comment by nishtha53** [comment]
Yes sure @anuragh2002 can work on content part. I will work on design of that page!

**Comment by Ankit-11525** [comment]
Yeah sure @anuragh2002

**Comment by nishtha53** [comment]
@anuragh2002 Once you are done with content you can provide me I will work on it.
